### ISSUE TITLE
Stylize error message using CSS

### ISSUE THREAD

**Issue by burntcarrot** [burntcarrot/peavote#2]
The error message displayed is in plaintext in nature. We can create a custom class for displaying the error message in a more informative way, just like [Bootstrap Alerts.](https://getbootstrap.com/docs/4.0/components/alerts)

Basic knowledge of CSS is required.

**How do I start?**
Run the app using [Docker-Compose](https://github.com/burntcarrot/peavote#run-locally).

**Comment by burntcarrot** [comment]
**This issue is open for Hacktoberfest 2021. Feel free to contribute!**

**Comment by shubhankit123** [comment]
is the issue still open ?

**Comment by moulikagrawal** [comment]
Please assign this issue to me, i want to work on it

**Comment by burntcarrot** [comment]
@itsdharmxd @shubhankit123 @moulikagrawal It's open. Since this issue requires some knowledge of React and CSS, I hope you can work on it.

As we are having multiple people willing to work on this issue, **I unfortunately would only accept the PR which can add the given feature appropriately as described.**

Hacktoberfest starts in less than 24 hours, you can make a PR after Hacktoberfest starts.

**Comment by itsdharmxd** [comment]
**@burntcarrot** I can see only one  "error"   in   **"peavote/client/src/pages/Home.js"**     line:129  ,
or i missed some ? please clarify

**Comment by burntcarrot** [comment]
@itsdharmxd Yes, that the error message used for displaying.

I suggest you to run the entire app using Docker-Compose, so that you can see the changes after adding the custom style using CSS.

**Comment by burntcarrot** [comment]
Also, you can change wrap the `<p>` tag inside a `div`, and use a class with that `div` for stylizing.

Example:

```html
{error &&
    <div class = "errorMessage>
          <p>{error}</p>
    </div>
}
```

and then add the `errorMessage` class with the style in [App.css](https://github.com/burntcarrot/peavote/blob/main/client/src/App.css).

**Comment by itsdharmxd** [comment]
![image](https://user-images.githubusercontent.com/68321391/135582341-ea27d80c-d95c-4cc9-9427-17155aad498b.png)

This if from inspect-mobile
![image](https://user-images.githubusercontent.com/68321391/135582437-b29683e8-bb26-4595-a2b4-824704147032.png)

should i give PR request any further improvement  you want

**Comment by burntcarrot** [comment]
Can you reduce the padding a little bit?

**Comment by itsdharmxd** [comment]
ya sure in 5 mins

**Comment by itsdharmxd** [comment]
![image](https://user-images.githubusercontent.com/68321391/135587947-d27dad64-3071-4164-a00a-2a06b9f5d37b.png)

![image](https://user-images.githubusercontent.com/68321391/135588016-a4e70cd2-38df-4402-b35b-032debbbc74c.png)
is this?

**Comment by burntcarrot** [comment]
Looks good! Please commit the changes so that I can merge the PR.

**Comment by itsdharmxd** [comment]
ya sure in 5 mins

**Comment by itsdharmxd** [comment]
i have given  PR with 2 commits @burntcarrot

**Comment by burntcarrot** [comment]
Closed in favor of #4.
